documentation/contributing/syncing_backporting/index.markdown

```diff
@@ -0,0 +1,205 @@
+---
+author: henningkayser
+comments: true
+date: 2021-11-05 07:00:00+01:00
+layout: contribution-guide
+slug: syncing
+title: Syncing and Backporting
+redirect_from: "/documentation/contributing/syncing_backporting.html"
+---
+
+# Syncing and Backporting
+
+In order to make bug-fixes and features available across repos and distros, we need to migrate them from time to time.
+Syncing refers to the process of preparing a merge commit from one branch to another that keeps the commit history intact.
+This allows us to better keep track of all the changes that need to get synced, but also helps with resolving issues later on since the original history of the source branch is available in the correct order for later review.
+We usually only do this from the MoveIt 1 master branch to the `main` branch of MoveIt 2, but other repos could also use the same strategy.
+
+Backporting is the act of migrating isolated pull reqests or commits from one branch to another using a cherry-pick.
+The typical use case is that we backport bug-fixes from the MoveIt 2 `main` branch to stable distro branches or to MoveIt 1.
+
+Below are step-by-step guides that should help maintainers to perform a proper sync or backport.
+
+## Syncing from MoveIt 1 to MoveIt 2
+
+Let's assume `origin` is the git remote for MoveIt 2 and `moveit1` the one for MoveIt 1, like below (using SSH works just the same, of course):
+
+
+**origin:** https://github.com/ros-planning/moveit2.git
+
+**moveit1:** https://github.com/ros-planning/moveit.git
+
+
+
+### 1. Create a new sync branch
+
+Fetch the latest MoveIt 2 `main` branch and use it for creating a new sync branch `pr-sync`:
+
+```
+git fetch origin
+git checkout -b pr-sync origin/main
```

Comment:
Does not justify an update, but in case you didn't know it already:

`git switch -c pr-sync origin/main`